srcpkgs/rmlint/template

```diff
@@ -12,6 +12,7 @@ homepage="http://rmlint.rtfd.org/"
 changelog="https://raw.githubusercontent.com/sahib/rmlint/master/CHANGELOG.md"
 distfiles="https://github.com/sahib/${pkgname}/archive/v${version}.tar.gz"
 checksum=10e72ba4dd9672d1b6519c0c94eae647c5069c7d11f1409a46e7011dd0c6b883
+patch_args="-Np1"
```

Comment:
Just remove the prefix from the patch: `a/lib/cfg.c -> lib/cfg.c`, then this isn't necessary.